src/parsers/ZoneDataParser.ts

```diff
@@ -1,12 +1,17 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import {Zone, zoneArray} from '../ps2alerts-constants/zone';
-import {injectable} from 'inversify';
+import {Injectable} from '@nestjs/common';
 import {censusOldFacilities} from '../ps2alerts-constants/censusOldFacilities';
 import ApplicationException from '../exceptions/ApplicationException';
 import {CensusFacilityRegion, CensusRegionResponseInterface} from '../interfaces/CensusRegionEndpointInterfaces';
-import {getZoneVersion} from '../utils/zoneVersion';
-
-@injectable()
+import * as IndarMap from '../ps2alerts-constants/maps/regions-2-1.0.json';
```

Comment:
Not quite sure what happened here but this needs undoing, the data is in Census now so we don't need these fallbacks